STYLE-GUIDE.md

```diff
@@ -2,22 +2,41 @@
 
 This document acts as a Style Guide for the language and wording used in exercises.
 
-## General principle
+## Application
 
-All content should be written in US English.
+These rules should be followed in all exercises. 
+Existing descriptions and test-cases can be updated to adhere to them without requiring "replacement" test cases.
+
+### Consistency within an exercise.
+
+There are some terms that have multiple valid spellings (e.g. "lower case" vs "lowercase"). 
+Where a consistent style has not been agreed within this document, these must be consistent within an exercise.
+
+### Exceptions
+
+Exercises may vary from these rules if there is general consensus across maintainers that this is reasonable. 
+For example, an exercise about converting between different units might choose not to use SI measurements.
+
+## Language
+
+All content should be written in US English. In the future other translations may occur, but the "official" Exercism language is US English.
+
+### Measurements
 All units of measurement must be [SI](https://en.wikipedia.org/wiki/International_System_of_Units) or [SI-derived](https://en.wikipedia.org/wiki/SI_derived_unit) units.
-In the future other translations may occur, but the "official" Exercism language is US English.
 
-## Explain or substitute mathematical and esoteric terms.
+### Abbreviations, acronyms and initialisms
+> All abbreviations, acronyms and initialisms should be defined. 
+> - If the unabbreviated term is clear, always include the full wording at first use (e.g. "Domain-specific language (DSL)");
+> - If the term needs more explaination to be clear to someone unfamiliar, explain it: (e.g. "The four basic types of SQL commands are Create, Read, Update, Delete. The initials of these come together to form the word CRUD.");
+> - If explaining the acryonm adds no tangible value to the student, simply introduce the acryonm as if it were a "proper noun" (e.g. "People's inherit DNA, a set of chemical instructions that influence how their bodies are constructed")
```

Comment:
```suggestion
> - If explaining the abbreviation adds no tangible value to the student, simply introduce it as if it were a "proper noun" (e.g. "People inherit DNA, a set of chemical instructions that influence how their bodies are constructed")
```

Did you mean to put this whole section in a blockquote too?

I think there should be an additional rule about avoiding jargon or abbreviations where another term would be fine (so don't talk about FIFOs if you could instead talk about queues; don't talk about things being RESTful if you can instead talk about what specific properties are good; don't talk about CRUD if you don't need to; don't use a jargon-heavy theme unless it really adds to the exercise; etc.)